Source/AtomicEditor/Application/AEEditorCommon.cpp

```diff
@@ -255,4 +258,53 @@ bool AEEditorCommon::ReadPreferences()
     return true;
 }
 
+void AEEditorCommon::ValidateWindow()
+{
+    Graphics* graphics = GetSubsystem<Graphics>();
+    IntVector2 windowPosition = graphics->GetWindowPosition();
+    int monitors = graphics->GetMonitorsNumber();
+    IntVector2 maxResolution;
+
+    for (int i = 0; i < monitors; i++)
+    {
+        IntVector2 monitorResolution = graphics->GetMonitorResolution(i);
+        maxResolution += monitorResolution;
+    }
+
+    if (windowPosition.x_ >= maxResolution.x_ || windowPosition.y_ >= maxResolution.y_ || windowPosition.x_ < 0 || windowPosition.y_ < 0)
+    {
```

Comment:
On OSX I just received: "Window was moved to -669,171" (I am thinking this is how it would be on Windows too), the position can go below zero and I am not sure how this is handled for monitors to the left of main display, in short I am not sure how to handle this one, perhaps absolute value or perhaps leave until it can be looked at closer